### DISCUSSION TITLE
2023/08/18/%E6%93%8D%E4%BD%9C%E7%B3%BB%E7%BB%9F%E8%BF%90%E8%A1%8C%E7%8E%AF%E5%A2%83/

### DISCUSSION THREAD

**Discussion by giscus** [ch1ny/ch1ny.github.io#66] [General]
# 2023/08/18/%E6%93%8D%E4%BD%9C%E7%B3%BB%E7%BB%9F%E8%BF%90%E8%A1%8C%E7%8E%AF%E5%A2%83/

https://qindarkstone.github.io/2023/08/18/%E6%93%8D%E4%BD%9C%E7%B3%BB%E7%BB%9F%E8%BF%90%E8%A1%8C%E7%8E%AF%E5%A2%83/

**Comment by qindarkstone** [comment]
mark一下
